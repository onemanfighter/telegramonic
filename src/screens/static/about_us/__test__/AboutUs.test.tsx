import { render } from '@testing-library/react';
import AboutUs from '../AboutUs';

describe('AboutUs screen', () => {
  it('should render correctly', () => {
    const { container } = render(<AboutUs />);

    expect(container).toMatchSnapshot();
  });
});
