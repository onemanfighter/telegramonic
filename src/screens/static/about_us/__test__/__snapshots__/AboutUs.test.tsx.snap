// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      About Us
    </h1>
  </div>
</div>
`;
